src/main/resources/public/jsx/geo-dash-widget-editor.js

```diff
@@ -891,69 +858,63 @@ class BasicLayout extends React.PureComponent{
-        </React.Fragment>
+            <button type="button" className="btn btn-primary" onClick={this.onCreateNewWidget} disabled={!this.state.formReady}>Create</button>
+        </React.Fragment>;
 
     }
     getBaseMapSelector(){
-        if(this.state.selectedWidgetType == "ImageCollection" || this.state.selectedWidgetType == "DualImageCollection" || this.state.selectedWidgetType == "imageAsset" || this.state.selectedWidgetType == "imageCollectionAsset") {
+        if(this.state.selectedWidgetType === "ImageCollection" || this.state.selectedWidgetType === "DualImageCollection" || this.state.selectedWidgetType === "imageAsset" || this.state.selectedWidgetType === "imageCollectionAsset" || this.state.selectedWidgetType === "ImageElevation") {
             return <React.Fragment>
                 <label htmlFor="widgetIndicesSelect">Basemap</label>
                 <select name="widgetIndicesSelect" value={this.state.WidgetBaseMap} className="form-control"
                         id="widgetIndicesSelect" onChange={this.onDataBaseMapSelectChanged}>
                     {this.baseMapOptions()}
                 </select>
-            </React.Fragment>
+            </React.Fragment>;
         }
-        else{return;}
     }
     baseMapOptions(){
-
-        let options = _.map(this.state.imagery, function (imagery) {
-            return <option key={imagery.id} value={imagery.id}> {imagery.title} </option>
+        return _.map(this.state.imagery, function (imagery) {
+            return <option key={imagery.id} value={imagery.id}> {imagery.title} </option>;
         });
-        return options;
     }
-    getDataType()
+    getDataTypeSelectionControl()
     {
-        if(this.state.selectedWidgetType == "-1")
+        if(this.state.selectedWidgetType === "-1")
         {
-            return
+            return <br/>;
         }
-        else if(this.state.selectedWidgetType == "statistics")
+        else if(this.state.selectedWidgetType === "statistics")
         {
-            if(this.state.FormReady != true){
+            if(this.state.formReady !== true){
                 this.setState({
-                    FormReady: true
+                    formReady: true
                 });
             }
             return <React.Fragment>
                 <div className="form-group">
                     <label htmlFor="widgetTitle">Title</label>
                     <input type="text" name="widgetTitle" id="widgetTitle" value={this.state.WidgetTitle} className="form-control" onChange={this.onWidgetTitleChange}/>
                 </div>
-            </React.Fragment>
+            </React.Fragment>;
         }
-        else if(this.state.selectedWidgetType == "imageAsset" || this.state.selectedWidgetType == "imageCollectionAsset")
+        else if(this.state.selectedWidgetType === "imageAsset" || this.state.selectedWidgetType === "imageCollectionAsset" || this.state.selectedWidgetType === "ImageElevation")
```

Comment:
["imageAsset", "imageCollectionAsset" "ImageElevation"].includes(this.state.selectedWidgetType)